.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,2 @@
+<!-- Love nox? Please consider supporting our collective:
+👉  https://opencollective.com/python-nox/donate -->
```

Comment:
We use the multiple issue template format

https://github.com/theacodes/nox/tree/master/.github/ISSUE_TEMPLATE